app/managers.py

```diff
@@ -0,0 +1,37 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema_db")

```

Comment:
db should have .sqlite3 extension

Reply:
@Bohdan-Salamakha 
<img width="616" alt="image" src="https://user-images.githubusercontent.com/77115535/236671861-9a8d5c58-1a55-483c-9de2-3312bd5b1561.png">
As I understand, I should create DB inside SQLiteStudio and then add it into a project, isn't it? If yes, I've done this, but when I add extension for the file, my program doesn't see the DB. To resolve your problem, I should have added the DB file into Git, but a checklist said, that I [shouldn't](https://github.com/mate-academy/py-actor-manager/blob/master/checklist.md#dont-push-db-files). So what do I need to do?

Reply:
cinema_db is a file it does not SQL database. Please read about extensions of SQLite
You should not add db file into git. Read about gitignore